ee/modules/140-user-authz/images/webhook/web/hook/handler.go

```diff
@@ -76,39 +93,35 @@ func (h *Handler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 
 func (h *Handler) authorizeNamespacedRequest(request *WebhookRequest, entry *DirectoryEntry) *WebhookRequest {
 	if !hasLimitedNamespaces(entry) {
-		// User has no namespaces restriction.
+		// User has no namespaced restriction.
```

Comment:
Hmm, it is hard to calculate all the changes in this method:

What I offer:
1. Mark `allowAccessToSystemNamespaces` deprecated as well. 
2. If a `namespaceSelector` is specified - only it works. Otherwise, use the old logic of limitNamespaces and allowAccessToSystemNamespaces.


Thus the `authorizeNamespacedRequest` method can be simplified to something like this.
```go
if !hasLimitedNamespaces(entry) {
  return request
}

if entry.NamespaceSelector != nil {
  return h.authorizeWithNamespaceSelector(request, entry)
}

... // original logic of the authorizeNamespacedRequest method is here without changes
```

P.S. `allowAccessToSystemNamespaces` is an obscure option, it is better to add a simple namespace selector to the example section for system namespaces filtration, e.g.,

```yaml
namespaceSelector:
  labelSelector:
    matchLabels:
       heritage: deckhouse
```